portfolio/src/main/webapp/js/script.js

```diff
@@ -12,59 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/********************************************************
- * Listens to the about link in the navigation bar and
- * turns the text to yellow when the about section is visited.  
+/**
+    Fetch demo
  */
-document.getElementById('about-link').addEventListener('click', () => {
-    let about = document.getElementById('about-link')
-    let contact = document.getElementById('contact-link')
-    let projects = document.getElementById('projects-link')
-
-    if(!about.classList.contains('yellow-text'))
-  	    about.classList.add('yellow-text')
-
-    if(contact.classList.contains('yellow-text'))
-  	    contact.classList.remove('yellow-text')
-
-    if(projects.classList.contains('yellow-text'))
-        projects.classList.remove('yellow-text') 
-})
-
-/********************************************************
- * Listens to the projects link in the navigation bar and
- * turns the text to yellow when the projects section is visited.  
- */
-document.getElementById('projects-link').addEventListener('click', () => {
-    let about = document.getElementById('about-link')
-    let contact = document.getElementById('contact-link')
-    let projects = document.getElementById('projects-link')
-
-    if(!projects.classList.contains('yellow-text'))
-        projects.classList.add('yellow-text')
-
-    if(contact.classList.contains('yellow-text'))
-        contact.classList.remove('yellow-text')
-
-    if(about.classList.contains('yellow-text'))
-        about.classList.remove('yellow-text')
-})
-
-/********************************************************
- * Listens to the contact link in the navigation bar and
- * turns the text to yellow when the contact section is visited.  
- */
-document.getElementById('contact-link').addEventListener('click', () => {
-    let about = document.getElementById('about-link')
-    let contact = document.getElementById('contact-link')
-    let projects = document.getElementById('projects-link')
-
-    if(!contact.classList.contains('yellow-text'))
-        contact.classList.add('yellow-text')
-
-    if(about.classList.contains('yellow-text'))
-        about.classList.remove('yellow-text')
-
-    if(projects.classList.contains('yellow-text'))
-        projects.classList.remove('yellow-text')
-})
+ function fetchData(){
+     fetch('/data')
+     .then((response) => response.text()
+     .then((text) => showData(text)), () => console.log("Failed to get informatin at /data"))
```

Comment:
Here it may not compile, please check the match between open and close parentheses.